requests/utils.py

```diff
@@ -548,7 +550,8 @@ def should_bypass_proxies(url):
     # exceptions we've seen, though: this call failing in other ways can reveal
     # legitimate problems.
     try:
-        bypass = proxy_bypass(netloc)
+        invalid_indicator = int(time.time() / 60)
+        bypass = _cached_proxy_bypass(netloc, invalid_indicator)
```

Comment:
You're right, I should move it outside of the try block to make the exception catching simple.
However, since we wrap the `proxy_bypass` by `_cached_proxy_bypass`, do you think we should put the try block here? Or should we put it inside the wrapper function?